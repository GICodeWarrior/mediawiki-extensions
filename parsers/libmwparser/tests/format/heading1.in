== heading <nowiki> ==
text
  </nowiki> text ==
== heading

</nowiki>

  text

  text <table> <td>
text
   text
   text </table>

   text
