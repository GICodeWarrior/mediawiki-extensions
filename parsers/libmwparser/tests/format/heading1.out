BEGIN ARTICLE
    BEGIN HEADING[2]
        SPACE[ ]
        WORD[heading]
        SPACE[ ]
        NOWIKI[==
text]
        END NOWIKI
        SPACE[ ]
        WORD[text]
        SPACE[ ]
    END HEADING
    BEGIN PARAGRAPH
        SPECIAL[=]
        SPECIAL[=]
        SPACE[ ]
        WORD[heading]
    END PARAGRAPH
    BEGIN PARAGRAPH
        SPECIAL[</]
        WORD[nowiki]
        SPECIAL[>]
    END PARAGRAPH
    BEGIN PRE
        WORD[text]
    END PRE
    BEGIN PARAGRAPH
        SPACE[  ]
        WORD[text]
        SPACE[ ]
    END PARAGRAPH
    SPACE[ ]
    BEGIN TABLE
        BEGIN TABLE BODY
            BEGIN ROW
                BEGIN CELL
                    BEGIN PARAGRAPH
                        WORD[text]
                    END PARAGRAPH
                    BEGIN PRE
                        WORD[text]
                    END PRE
                    BEGIN PARAGRAPH
                        SPACE[   ]
                        WORD[text]
                        SPACE[ ]
                    END PARAGRAPH
                END CELL
            END ROW
        END TABLE BODY
    END TABLE
    BEGIN PRE
        WORD[text]
    END PRE
END ARTICLE
